packages/clay-toolbar/src/index.tsx

```diff
@@ -52,6 +60,7 @@ const ClayToolbar: React.FunctionComponent<IProps> & {
 		{
 			'component-tbar': !subnav,
 			'subnav-tbar': !!subnav,
+			'tbar-light': !!light,
```

Comment:
I think that I got confused a bit by seeing `!!subav` but that has double bangs because that's not a simple prop. You're right, it's unnecessary to have them for `light`

Reply:
You would think that this is the kind of thing that the TS compiler would be able to warn about (casting an already-boolean to a boolean); maybe in a future "strict" mode update...

Reply:
Yea, wouldn't that be nice